stl/inc/xutility

```diff
@@ -3261,6 +3261,23 @@ namespace ranges {
     // VARIABLE ranges::prev
     inline constexpr _Prev_fn prev{_Not_quite_object::_Construct_tag{}};
 
+    // FUNCTION TEMPLATE _Find_last_iterator
+    template <forward_iterator _It, sentinel_for<_It> _Se>
+    _NODISCARD constexpr _It _Find_last_iterator(
+        const _It& _First, const _Se& _Last, const iter_difference_t<_It> _Count) {
+        // Find the iterator in [_First, _Last) (of length _Count) which equals _Last
+        _STL_INTERNAL_CHECK(_RANGES distance(_First, _Last) == _Count);
+        (void) _First;
+        (void) _Last;
+        (void) _Count;

```

Comment:
Were these added preemptively or are they necessary to silence compiler warnings? We were able to purge almost all occurrences of this; only a few are still affected by the compiler warning/bug.

Reply:
Preemptively; I'll verify.

Reply:
The test passes with these removed; I need to retrain my `if constexpr` reflexes.